Add support for Basilisp file extension to Calva Spritz (#2628)
BetterThanTomorrow/calva: CHANGELOG.md

## [Unreleased]

> - [Sync the extensions between Calva an Calva Spritz](https://github.com/BetterThanTomorrow/calva/issues/2629)

Reviewer: ```suggestion
- [Synchronize the file extensions for Calva and Calva Spritz](https://github.com/BetterThanTomorrow/calva/issues/2629)
```
